public/app/features/trails/MetricSelect/MetricSelectScene.tsx

```diff
@@ -521,9 +527,11 @@ export class MetricSelectScene extends SceneObjectBase<MetricSelectSceneState> i
 
     const blockingMessage = isLoading
       ? undefined
-      : (noMetrics && 'There are no results found. Try a different time range or a different data source.') ||
-        (tooStrict && 'There are no results found. Try adjusting your search or filters.') ||
-        undefined;
+      : missingOtelTargets
+        ? 'There are no metrics found. Please adjust your filters based on your OTel resource attributes.'
+        : (noMetrics && 'There are no results found. Try a different time range or a different data source.') ||
+          (tooStrict && 'There are no results found. Try adjusting your search or filters.') ||
```

Comment:
Nice! Seeing this `tooStrict` condition makes me want to expand on the messaging introduced here:

https://github.com/grafana/grafana/pull/98775/files#diff-2b75a71fd9f9b0d24f91835cfdbbad4484f15063fb6c59a616c1f9ab3812e8b6